NFIQ2/NFIQ2Algorithm/src/tool/nfiq2_ui_refresh.cpp

```diff
@@ -302,9 +304,15 @@ NFIQ2UI::coreCompute(const NFIQ::FingerprintImageData &wrappedImage,
 	std::list<NFIQ::QualityFeatureData> featureVector;
 	std::list<NFIQ::QualityFeatureSpeed> featureTimings;
 
-	const unsigned int qualityScore = model.computeQualityScore(
-	    wrappedImage, true, actionableQuality, true, featureVector, true,
-	    featureTimings);
+	unsigned int qualityScore = 255;
+
+	try {
+		qualityScore = model.computeQualityScore(wrappedImage, true,
+		    actionableQuality, true, featureVector, true,
+		    featureTimings);
+	} catch (const NFIQ::NFIQException &e) {
+		throw e;
```

Comment:
Don't really need to catch and rethrow, just don't catch. `qualityScore` could be `const` and doesn't need a magic `255` if so.